internal/report/github/summary.go

```diff
@@ -0,0 +1,127 @@
+package github
+
+import (
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/saucelabs/saucectl/internal/report"
+)
+
+type Reporter struct {
+	beginTime       time.Time
```

Comment:
Optional: Think it's more conventional to say `startTime`? Up to you.
```suggestion
	startTime       time.Time
```

Reply:
I agree ! Updated !